### PR TITLE
Deprecated object-to-vuex-store in favor of Tuex.

### PR DESCRIPTION
**Pull request by Raiondesu**
Due to a relatively high initial download count I decided to push the idea further with Tuex.
Tuex wiki is [in progress](https://github.com/Raiondesu/Tuex/wiki), by the way.

### CODE REVIEW

**Comment by Raiondesu** [comment]
Tuex wiki is in progress.

File: README.md [hunk 1]
```diff
@@ -1549,6 +1549,7 @@ the amazing Vue.js.
 ### State Management
 
  - [vuex](https://github.com/vuejs/vuex) - Centralized State Management for Vue.js.
+ - [tuex](https://github.com/Raiondesu/Tuex) - A mostly reasonable alternative to Vuex.
```
Comment (kazupon) [inline README.md]: >Always add your items to the end of a list. To be fair, the order is first-come-first-serve.
https://github.com/vuejs/awesome-vue/blob/master/CONTRIBUTING.md

`CONTRIBUTING.md` have a rule in the above.
Could you fix the it please?

Comment (Raiondesu) [reply README.md]: Oh, of course.

**Comment by Raiondesu** [comment]
@kazupon, changes respectively added.

**Pull request merged**